features/administration.feature

```diff
@@ -11,9 +11,13 @@ Feature: Administrate pupils, classes and teachers
 		When an admin puts in <firstName> and <secondName> and <email> of the new pupil
 		Then the admin should see new pupil with email <email> among his pupils
 		Then the teacher can manually submit a consent <email>
-		Then new pupil can log in
-		Then new pupil accepts data protection policy and sets new password for the profile
+		And log out
+		And go from start page to login page
+		And log in with <username> and <password>
```

Comment:
who is log in?

Reply:
is now changed to:
When admin logs in with email <adminsUsername> and password <password>
		Then admin goes to administration
		And an admin puts in <firstName> and <secondName> and <email> of the new pupil
		And the admin should see new pupil with email <email> among his pupils
		And admin manually submits a consent <email>
		And admin logs out